calcom/cal.com
# Button to test a workflow action

## What does this PR do?

Adds a button to test a workflow action by sending out an email or SMS. Additionally, some design improvements have been made.

Also fixes: #3963

Old design: 
<img width="834" alt="Screenshot 2022-08-16 at 15 11 11" src="https://user-images.githubusercontent.com/30310907/184962597-e41727a4-211e-4e61-bd50-1eb14f8c1a56.png">

New design: 
<img width="832" alt="Screenshot 2022-08-16 at 15 10 41" src="https://user-images.githubusercontent.com/30310907/184962501-f7089fee-19d0-409f-ae71-977f1d56b9fc.png">

Fixes #3745

 Loom Video: https://www.loom.com/share/f3db501b545a41cc8807e267f886d0f5

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

- Setup Sendgrid and Twilio to test locally: https://github.com/calcom/cal.com#workflows
- Create a workflow
- Choose any action (except send SMS to attendee)
- Click ‘Test Action’
- Choose reminder template or create your own custom template
- Check if you received the correct SMS or Email

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
